lib/rules/web/ocm_console/cluster_detail_machinepool_tab.xyaml

```diff
@@ -121,4 +123,41 @@ click_scale_machine_pool_button:
     element:
         selector:
             xpath: //li/button[text()='Scale']
+        op: click
+check_edit_taints_in_dropdown:
+    action: expand_actions_of_machine_pool
+    element:
+        selector:
+            xpath: //button[text()='Edit taints']
+click_edit_taints_in_dropdown:
+    element:
+        selector:
+            xpath: //button[text()='Edit taints']
+        op: click
+check_no_default_in_dropdown_when_editing_taints:
+    elements:
+        - selector:
+            xpath: //select[@id='machinePoolId']
+          op: click
+        - selector:
+            xpath: //option[text()='Default']
+          missing: true
+refresh_on_machine_pool_page:
+    element:
+        selector:
+            xpath: //button[@aria-label='Refresh']
+        op: click
+
+
+
+        
```

Comment:
Too many space lines at here.

Reply:
Yes, whitespace lines should be deleted.